product/repository/cs-studio.product

```diff
@@ -21,7 +21,8 @@
 -Dosgi.configuration.cascaded=true
 -Dosgi.sharedConfiguration.area.readOnly=true
 -Dosgi.install.area.readOnly=true
--Dosgi.requiredJavaVersion=11
+-Dosgi.requiredJavaVersion=17
+-Djdk.util.zip.disableZip64ExtraFieldValidation=true
```

Comment:
I didn't check, but we might want similar flags on some of the other products too (`alarm-server`, `archive-engine` etc).

Reply:
I haven't added these. I was able to start the CSS, open OPI in OPI editor, make changes, and see the OPIs working in the IBEX GUI